docscan/session/retrieve/supplementary_document_text_extraction_task_response.go

```diff
@@ -0,0 +1,11 @@
+package retrieve
+
+// SupplementaryDocumentTextExtractionTaskResponse represents a Supplementary Text Extraction task response
```

Comment:
Not sure if you intentionally missed this out?
```suggestion
// SupplementaryDocumentTextExtractionTaskResponse represents a Supplementary Document Text Extraction task response
```

Reply:
Corrected in 403212f